docs/Developers/Writing_Native_Bears.rst

```diff
@@ -519,5 +519,71 @@ can be used as follows:
         ASCIINEMA_URL = 'https://asciinema.org/a/80761'
         SEE_MORE = 'https://www.pylint.org'
 
+Aspect Bear
+-----------
+
+Aspect is a feature in coala that make configuring coala in project more easy
+and language agnostic. For more detail about aspect, see cEP-0005 in
+https://github.com/coala/cEPs/blob/master/cEP-0005.md.
+
+An aspect-compliant bear MUST:
+
+1. Declare list of aspect it can fix and detected. Note that the aspect MUST be
+   a leaf aspect. You can see list of supported aspect here
+   https://github.com/coala/aspect-docs.
+2. Declare list of supported language. See list of supported language
+   https://github.com/coala/coala/tree/master/coalib/bearlib/languages/definitions.
+3. Map setting to its equivalent aspect or taste using ``map_setting_to_aspect``
+   decorator.
+4. Yield result with relevant aspect.
+
+For example, let's make an aspect bear named SpellingCheckBear.
+
+.. code:: python
+
+    from coalib.bearlib.aspects import map_setting_to_aspect
+    from coalib.bearlib.aspects.Spelling import (
+        DictionarySpelling,
+        OrgSpecificWordSpelling,
+    )
+    from coalib.bears.LocalBear import LocalBear
+
+    class SpellingCheckBear(
+            LocalBear,
+            aspect={
+                'detect': [DictionarySpelling, OrgSpecificWordSpelling]
+            },
+            languages=['Python']):
+
+        @map_setting_to_aspect(
+            use_standard_dictionary=DictionarySpelling
+            additional_dictionary_words=OrgSpecificWordSpelling.specific_word
+        )
+        def run(self,
+                filename,
+                file,
+                use_standard_dictionary:bool=True,
+                additional_dictionary_words:list=None):
+            """
+            Detect wrong spelling.
+
+            :param use_standard_dictionary:     Use standard English dictionary.
+            :param additional_dictionary_words: Additional list of word.
+            """
+            # Imagine this is where we save our standard dictionary.
+            dictionary_words = ['lorem', 'ipsum']
+
+            if not use_standard_dictionary:
+                dictionary_words = []
+            dictionary_words += additional_dictionary_words
+
+            for word in file.split():
+                if word not in dictionary_words:
+                    yield self.new_result(
+                        message='Wrong spelling',
+                        additional_info=(DictionarySpelling.docs
```

Comment:
shouldnt `DictionarySpelling.docs.importance_reason` be the default value for `additional_info`?

Reply:
Yes it should be. If you mean
 
> If result have `aspect` arg, it should be defaulting to use `aspect.docs.importance_reason`

then, no, it is not implemented yet.

Reply:
Is there a bug about it?!  If not, please create one!
This would fall in your main project scope, of making it easy to use aspects.

Reply:
Created https://github.com/coala/coala/issues/4680

Reply:
That issue is included in this PR too.